### PR TITLE
fix tables and lists styles

### PR DESCRIPTION
**Pull request by artemilchuk**

### CODE REVIEW

File: source/app/scss/base/_lists.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ul,
 ol {
-  list-style-type: none;
-  margin: 0;
+  margin: 0 0 0 25px;
```
Comment (chudaol) [inline source/app/scss/base/_lists.scss:3]: wouldn't it break all our lists?

File: source/app/scss/base/_tables.scss [hunk 2]
```diff
@@ -7,15 +7,15 @@ table {
...
 
 th {
-  border-bottom: 1px solid darken($base-border-color, 15%);
   font-weight: 600;
   padding: $small-spacing 0;
   text-align: left;
 }
 
 td {
+  border-bottom: 1px solid darken($base-border-color, 15%);
```
Comment (chudaol) [inline source/app/scss/base/_tables.scss:16]: will it change all the tables on platform?

**Comment by chudaol** [comment]
can you please also tell where it can be tested? on which page? thanks

**Comment by artemilchuk** [comment]
You can test this in "Export course"
https://courses.ed-era.com/courses/EdEra/ib102/IB102/courseware/fa0a63fe797f47bfbd2cf78f70876f82/617edb34c8214c55bfe79b687c680d4d/

Also, it will change tables only in courseware

File: source/app/scss/base/_lists.scss (hunk 1 above)
Comment (chudaol) [reply source/app/scss/base/_lists.scss:3]: also i just tested it and it looks a bit cleaner without margin at all

Comment (artemilchuk) [reply source/app/scss/base/_lists.scss:3]: I tested it and have some bugs :-(

image

**Pull request merged**